### DISCUSSION TITLE
Collapse using checkbox, onBlur works?

### DISCUSSION THREAD

**Discussion by MacielFede** [saadeghi/daisyui#1805] [Q&A] [answered]
Hi, for context, I'm in a React project developing a sidebar that contains `collapse` elements and inside them are links.
I'm using the `checkbox` variant but I want to know if there's a way to display only 1 of the `collapse` at a time. 
Essentially I want the `collapse` that's open to close when loses focus without using the `collapse that uses focus`, because the links don't work in that variant.

**Comment by saadeghi** [comment, marked as answer]
In daisyUI `3.0` (Will be ready by the end of this month) there will be new styles for `radio` inputs inside `collapse` as well so all other items will be closed automatically (because unlike checkbox only one radio can be active)  
But until daisyUI `3.0` gets ready, you can control the checkbox values using JS. Something like this:  
https://stackblitz.com/edit/daisyui-react-vite-6jqorn?file=src%2FApp.jsx
